modules/ossm-observability-cli.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//* service_mesh/v2x/ossm-observability.adoc
+
+
+[id="ossm-observability-cli_{context}"]
+= Accessing metrics and tracing data from the CLI
+
+Access the Jaeger, Prometheus, and Grafana consoles to view and manage your data. 
+
+.Procedure
+
+. Run the following command to switch to the control plane project. In this example, `istio-system` is the control plane project.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
+
+. Run this command from the CLI to get the routes to {ProductName} components.
++
+[source,terminal]
+----
+$ oc get routes
+----
++
+This command returns URLs for the web consoles of Kiali, Jaeger, Prometheus, and Grafana, and any other routes in your service mesh.
+
+. Copy the URL from the `HOST/PORT` column into a browser to open the console. 
```

Comment:
Should this say "the appropriate URL" or something similar? Previous sentence talks about several URLs and then this refers to one specific one.